server/views/pages/link/emailSent.njk

```diff
@@ -0,0 +1,78 @@
+{% extends "layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/fieldset/macro.njk" import govukFieldset %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
+{% set pageTitle = applicationName + " - Request a link" %}
+{% set pageId = 'email-sent' %}
+
+{% block content %}
+
+  <main class="app-container govuk-body">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl">Now check your emails</h1>
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+
+        {% set html %}
+          <p class="govuk-notification-banner__heading">We've sent a link to your email - {{ emailSentTo }}</p>
+        {% endset %}
+
+        {{ govukNotificationBanner({
+          type: 'success',
+          disableAutoFocus: true,
+          html: html
+        }) }}
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <h3>What to do if you don't get the link</h3>
+        <p>Check that the:
+          <ul>
+            <li>email address you've entered is correct - it needs to be valid for the Criminal Justice Secure Mail (CJSM) service</li>
+            <li>link hasn't gone to your 'spam' or 'junk' folder
+          </ul>
+        </p>
+      </div>
+    </div>
+
+	<div class="govuk-grid-row">
+	  <div class="govuk-grid-column-two-thirds">
+		<form id="email-form" action="request-link" method="post">
+		  <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+
+		  {% call govukFieldset({ }) %}
+			{{ govukInput({
```

Comment:
We could write a cheerio unit test here for quicker feedback on page rendering? Just check that the input field and submit button are displayed and have the correct types? It's such a trivial screen now sure it's worth it.